nbgitpuller/static/js/gitsyncview.js

```diff
@@ -64,9 +65,18 @@ export class GitSyncView{
         }
     }
 
-    setRecoveryLink(isError) {
+    setContainerError(isError, errorText='') {
         if (isError) {
-            this.recovery.classList.toggle('hidden', !visible);
+            this.containerError.classList.toggle('hidden', !this.visible);
+        }
+        const button = this.copyError;
+        button.onclick = async () => {
+            try {
+                await navigator.clipboard.writeText(errorText);
```

Comment:
Thinking about what other thing would be useful, can you also copy the nbgitpuller URL itself? that contains important and helpful info about the repo and branch that I think help a lot. But let's just copy the url - base_url (so just /git-sync?query-params) so we don't automatically include the username here, as that could be PII.